CAP/examples/DiscreteSkeletalFiniteCategory.g

```diff
@@ -0,0 +1,39 @@
+#! @Chapter Examples and Tests
+
+#! @Section Discrete skeletal finite category
+
+#! @Example
+LoadPackage("CAP", false );
+#! true
+gap_objects := [ 1..5 ];;
```

Comment:
Could you please write `[ 1 .. 5 ] ` instead of `[ 1..5 ]`. The later would not translate properly to Julia.